modules/cluster-wide-proxy-updates.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+//OSDOCS-3001 Configuration for cluster-wide-proxy Day 2
+:_content-type: PROCEDURE
+[id="cluster-wide-proxy-updates_{context}"]
+= Configuring or updating your cluster-wide proxy after installation
+
+ifdef::openshift-rosa[]
+As the cluster owner, you may wish to add a proxy to your created cluster after installation, or you may wish to make changes to your proxy that you configured during cluster installation. The `rosa` CLI provides some options for adding a proxy to your cluster or modifying an existing proxy on your cluster.
+endif::[]
+ifdef::openshift-dedicated[]
+As the cluster owner, you may wish to add a proxy to your created cluster after installation, or you may wish to make changes to your proxy that you configured during cluster installation. The `ocm` CLI provides some options for adding a proxy to your cluster or modifying an existing proxy on your cluster.
+endif::[]
+
+You may need to perform these actions if:
+
+* the cluster-wide proxy is configured after installation,
+* the proxy's network address needs to be updated, or
+* any of the proxy's certificate authorities have expired and the additional trust bundle needs to be replaced.
+
+[NOTE]
+====
+The cluster applies the configuration to the cluster’s control plane and worker nodes. This process results in each node in the cluster temporarily being placed into an unschedulable state and drained of its workloads while applying the configuration. Each node will be restarted as part of this process.
+====
+
+.Prerequsites
+ifdef::openshift-rosa[]
+* You have the `rosa` CLI installed and configured.
+endif::[]
+ifdef::openshift-dedicated[]
+* You have the `ocm` CLI installed and configured.
+endif::[]
+
+.Procedure
+* To edit a cluster, run the following command:
++
+ifdef::openshift-rosa[]
+[source,terminal]
+----
+$ rosa edit cluster \
+ --cluster $CLUSTER_NAME \
+ --additional-trust-bundle-file $CA_BUNDLE_FILE \
+ --http-proxy $HTTP_PROXY \
+ --https-proxy $HTTPS_PROXY
+----
+endif::[]
+ifdef::openshift-dedicated[]
+[source,terminal]
+----
+$ ocm edit cluster \
+ --cluster $CLUSTER_NAME \
+ --additional-trust-bundle-file $CA_BUNDLE_FILE \
+ --http-proxy $HTTP_PROXY \
+ --https-proxy $HTTPS_PROXY
+----
+endif::[]
++
+While the `additional-trust-bundle-file`, `http-proxy`, and `https-proxy` arguments are optional, if you set a `additional-trust-bundle-file` without either an `http-proxy` or `https-proxy` argument, then the additional trust bundle will still be used for verifying cluster system egress traffic.
+* You can verify that the proxy and certificate authority configuration updates have been successfully applied to your cluster by:
+** All of the MachineConfigPools are updated. Run the following command to see their status:
++
+[source,terminal]
+----
+$ oc get machineconfigpools
+----
++
+.Sample Output
+[source,terminal]
+----
+$ NEED EXAMPLE
```

Comment:
Here is an example, though its quite wide!

The important thing in the example is the "Updated" column which says "true"

```
oc get machineconfigpools 
NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master   rendered-master-d9a03f612a432095dcde6dcf44597d90   True      False      False      3              3                   3                     0                      31h
worker   rendered-worker-f6827a4efe21e155c25c21b43c46f65e   True      False      False      6              6                   6                     0                      31h
```